client/src/components/footer/footer.component.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ErrorBoundary from '../common/errorBoundary/errorBoundary.component';
+import styles from './footer.css';
+
+export default function createFooter() {
+  return (
+    <ErrorBoundary>
```

Comment:
ErrorBoundry seems redundant here since no props or methods could disrupt the work of this component.